portfolio/src/main/webapp/intro.html

```diff
@@ -3,12 +3,12 @@
   <head>
     <meta charset="UTF-8">
     <title>Sarah Zou's STEP Portfolio</title>
+    <link rel="stylesheet" href="style.css" type= 'text/css'>
```

Comment:
You don't need to specify `type` on `<link rel="stylesheet">`; it defaults to `text/css`.